docs/administration/configuring-replication/create-replication-rules.md

```diff
@@ -8,15 +8,15 @@ A replication endpoint must exist before you create a replication rule. To creat
 1. Log in to the Harbor interface with an account that has Harbor system administrator privileges.
 1. Expand **Administration**, and select **Replications**.
 
-   ![Add a replication rule](../../../img/replication-rule1.png)
+   ![Add a replication rule](../../img/replication-rule1.png)
 1. Click **New Replication Rule**.
 1. Provide a name and description for the replication rule.
 1. Select **Push-based** or **Pull-based** replication, depending on whether you want to replicate images to or from the remote registry.
 
-   ![Replication mode](../../../img/replication-rule2.png)
+   ![Replication mode](../../img/replication-rule2.png)
 1. For **Source resource filter**, identify the images to replicate.  
 
-   ![Replication filters](../../../img/replication-rule3.png)
+   ![Replication filters](../../img/replication-rule3.png)
```

Comment:
../../../ should be correct, it should be one more level than the actual folder structure, the image has its own directory basically, please double check